index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0">
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+       <div class="logo"><img src="images/image_geometry_1.png" alt="logo image"></div>
```

Comment:
You have made great efforts so far on this project. However, kindly implement the use of CSS background image for the images as requested by the previous reviewers. This is because they're only a design and not part of the content of the page.